Cargo.toml

```diff
@@ -9,6 +9,7 @@ repository = "https://github.com/filipdulic/bus-queue"
 readme = "README.md"
 
 [dependencies]
+lockfree = "*"
```

Comment:
Replaced std::sync::mpsc with lockfree::channel::mpsc to enable Sync trait on Waker and Sleeper in lib.rs